spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  before(:each) do
+    @facility = FacilityFactory.new
+    @or_facilities = DmvDataService.new.or_dmv_office_locations
+    @ny_facilities = DmvDataService.new.ny_dmv_office_locations
+    @mo_facilities = DmvDataService.new.mo_dmv_office_locations
+  end
+
+  describe "initialize" do
+    it "can itialize" do
+      expect(@facility).to be_an_instance_of(FacilityFactory)
+    end
+  end
+
+  describe "create_facility_or" do
+    it "creates Oregons facility" do
+      expect(@facility.create_facility_or(@or_facilities)).to all(be_an_instance_of Facility)
```

Comment:
This is good. I might also add some additional expects here
```ruby
expect(@facility.create_facility_or(@or_facilities)).to be_an(Array)
expect(@facility.create_facility_or(@or_facilities).first.address).to be_a(String)
expect(@facility.create_facility_or(@or_facilities).first.name).to be_a(String)
...
```
Do this for each test below as well.